pkg/sql/sqlbase/structured.proto

```diff
@@ -129,6 +129,19 @@ message ColumnDescriptor {
   // Expression to use to compute the value of this column if this is a
   // computed column.
   optional string compute_expr = 11;
+  // LogicalColumnID is lazily set, do not access directly,
+  // access using GetLogicalColumnID().
+  // LogicalColumnID is used to order columns visually. This is because
```

Comment:
I think the example here could be shrunk to just mention ordering in catalog tables, rather than the specific case.